modules/home/assets/js/components/TopSection.js

```diff
@@ -0,0 +1,36 @@
+import { Box, Paper } from '@elementor/ui';
+import Typography from '@elementor/ui/Typography';
+import Button from '@elementor/ui/Button';
+import YoutubeIcon from './icons/youtube-icon';
+const TopSection = ({topScreenProps}) => {
+	return (
+		<Paper justifyContent="space-between" alignItems="flex-start" gap={ 4 } elevation={ 0 } sx={ { display: 'flex', flexDirection: 'row', bgcolor: 'theme.palette.background.default', width: '80%', marginX: 'auto', mt: 6 } }>
```

Comment:
I think for consistency `display flex` should be outside of the `sx`.
Width: there are components that have max-width presets, like `sm` and `xl`. I think that is preferred over 80%. And what does that mean responsively?

Maybe... <container> would be better, because it has the max-width. In that case, I don't think we need to add a MarginX auto. Or do we get issues with the elevation effect?

alignItems="flex-start": isn't that default?
justifyContent="space-between" > what happens if we just use the gap?
gap={ 4 } > what happens responsively on smaller screen sizes?